14/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/block-editor/block-list-editor.md

```diff
@@ -246,187 +246,3 @@ If you know the Block List Editor only uses a single block, you can cast the col
 ## Build a Custom Backoffice View
 
 Building Custom Views for Block representations in Backoffice is the same for all Block Editors. [Read about building a Custom View for Blocks here](build-custom-view-for-blocks.md)
-
-## Creating BlockList programmatically
```

Comment:
removing this as the example does not make much sense and it also follows a bad practice of using a view instead of a class.